libautonomi/Cargo.toml

```diff
@@ -15,6 +15,7 @@ rmp-serde = "1.1.1"
 self_encryption = "0.29.2"
 serde = { version = "1.0.133", features = ["derive", "rc"] }
 sn_client = { path = "../sn_client" }
+sn_networking = { path = "../sn_networking" }
```

Comment:
What is this dependency needed for? I think we will end up depending on it down the road.

Reply:
Shouldn't have been added in this commit. No longer needed it after a refactor 😅